### ISSUE TITLE
no vehicles to lock nearby

### ISSUE THREAD

**Issue by Nikolovski777** [sprixGG/esx_carlock#5]
when i try to lock the car it says no vehicles no lock nearby
help?

**Comment by gaby** [comment]
I'm also having the same issue, even after using the code from #4

**Comment by gaby** [comment]
Nevermind, I just updated my garage/vehicle shop to the Advanced ones and now these changes WORK 👍

**Comment by HometownRP** [comment]
Have the same prob

**Comment by Splati** [comment]
I have the advanced garage but also the error

**Comment by TommieTankEngine** [comment]
Update code to use the latest esx version. 
https://github.com/sprixGG/esx_carlock/pull/4/commits/719164cd6d60cbe3b87ca9549cd25e65fff60506

**Comment by a3rgames** [comment]
thank you @TommieTankEngine

**Comment by JuampaZC** [comment]
It worked for me but when I lock a vehicle the license plate appears "null"

> Update code to use the latest esx version. [719164c](https://github.com/sprixGG/esx_carlock/commit/719164cd6d60cbe3b87ca9549cd25e65fff60506)

**Comment by TommieTankEngine** [comment]
> It worked for me but when I lock a vehicle the license plate appears "null"
> 
> > Update code to use the latest esx version. [719164c](https://github.com/sprixGG/esx_carlock/commit/719164cd6d60cbe3b87ca9549cd25e65fff60506)

Does the vehicle still lock?

**Comment by JuampaZC** [comment]
It works and it locks, what I mean is that in the message below on the left it appears "you opened your NULL vehicle"

> > It worked for me but when I lock a vehicle the license plate appears "null"
> > > Update code to use the latest esx version. [719164c](https://github.com/sprixGG/esx_carlock/commit/719164cd6d60cbe3b87ca9549cd25e65fff60506)
> 
> Does the vehicle still lock?

**Comment by TommieTankEngine** [comment]
> It works and it locks, what I mean is that in the message below on the left it appears "you opened your NULL vehicle"
> 
> > > It worked for me but when I lock a vehicle the license plate appears "null"
> > > > Update code to use the latest esx version. [719164c](https://github.com/sprixGG/esx_carlock/commit/719164cd6d60cbe3b87ca9549cd25e65fff60506)
> > 
> > 
> > Does the vehicle still lock?

This is not a issue with this script that is a issue with the car you are using with the script. I personally would remove the vehicle name.

**Comment by TommieTankEngine** [comment]
ESX.ShowNotification('You have ~r~locked~s~ your ~y~vehicle~s~.')
ESX.ShowNotification('You have ~r~unlocked~s~ your ~y~vehicle~s~.')

**Comment by JuampaZC** [comment]
In mine it says:
ESX.ShowNotification('You have ~r~locked~s~ your ~y~'..vehicleLabel..'~s~.')
ESX.ShowNotification('You have ~g~unlocked~s~ your ~y~'..vehicleLabel..'~s~.')

> ESX.ShowNotification('You have ~r~locked~s~ your ~y~vehicle~s~.') ESX.ShowNotification('You have ~r~unlocked~s~ your ~y~vehicle~s~.')

**Comment by TommieTankEngine** [comment]
Yes. That Is correct! But I would instead recommend that you use the two I previously posted so you can avoid any problems with the "NULL" vehicle.

**Comment by JuampaZC** [comment]
Thanks

> Yes. That Is correct! But I would instead recommend that you use the two I previously posted so you can avoid any problems with the "NULL" vehicle.

**Comment by MooseJawMike77** [comment]
I am using the newest esx legacy and I have tried updating the code but it still doesn't work. I keep getting a server callback error stating "carlock:isVehicleOwner" does not exist and it is telling to check server file for errors. What errors would I be looking for?
